internal/acceptance/service_principal_test.go

```diff
@@ -14,6 +21,76 @@ const awsSpn = `resource "databricks_service_principal" "this" {
 	display_name = "SPN {var.RANDOM}"
 }`
 
+func TestAccServicePrincipalHomeDeleteSuccess(t *testing.T) {
+	appId := "12345a67-8b9c-0d1e-23fa-4567b89cde04"
+	appId2 := "22345a67-8b9c-0d1e-23fa-4567b89cde04"
+	os.Setenv("appId", appId)
+	os.Setenv("appId2", appId2)
+	workspaceLevel(t, step{
+		Template: `
+			resource "databricks_service_principal" "a" {
+				application_id = "{env.appId}"
```

Comment:
```suggestion
				application_id = "{var.RANDOM_UUID}"
```

you can use the random UUID here